conda_smithy/templates/conda-forge.yml.ci-skel.tmpl

```diff
@@ -0,0 +1,11 @@
+recipe_dir: {{ recipe_directory }}
+skip_render:
+  - README.md
+  - LICENSE.txt
+  - .gitattributes
+  - .gitignore
+  - build-locally.py
```

Comment:
Why `build-locally.py`? Might be fine, but I'm curious as to what the thought process was here.

Reply:
I left out build locally because the name is too generic as it doesn't include anything about the CI systems or conda-forge. If folks know that it is safe to render in they can always remove this line and get that file